stable/atlantis/README.md

```diff
@@ -47,11 +50,15 @@ The following options are supported.  See [values.yaml](values.yaml) for more de
 | `gitlab.hostname`                           | GitLab Enterprise only: The hostname of your GitLab Enterprise installation.                                                                                                                                                                                                                                                                               | n/a                    |
+| `logLevel`                                  | The level to use for logging.                             | n/a |
 | `orgWhiteList`                              | A whitelist of repositories from which Atlantis will accept webhooks. **This value must be changed for Atlantis to function correctly.**  Accepts wildcard characters (`*`).  Multiple values may be comma-separated.                                                                                                                                      | `github.com/yourorg/*` |
 | `serviceAccount.create` | Whether to create a Kubernetes ServiceAccount if no account matching `serviceAccount.name` exists. | `true` |
 | `serviceAccount.name` | The name of the Kubernetes ServiceAccount under which Atlantis should run.<br /><br />If no value is specified and `serviceAccount.create` is `true`, Atlantis will be run under a ServiceAccount whose name is the FullName of the Helm chart's instance.  <br /><br />If no value is specified and `serviceAccount.create` is `false`, Atlantis will be run under the `default` ServiceAccount. | n/a |
 | `serviceAccountSecrets.credentials`         | JSON object representing secrets for a Google Cloud Platform production service account.  Only applicable if hosting Atlantis on GKE.                                                                                                                                                                                                                      | n/a                    |
 | `serviceAccountSecrets.credentials-staging` | JSON object representing secrets for a Google Cloud Platform staging,service account.  Only applicable if hosting Atlantis on GKE.                                                                                                                                                                                                                          | n/a                    |
+| `service.port` | Specifies the port of the service.              | `80`                    |
+| `service.loadBalancerSourceRanges` | IP restrictions for the service load balancer if specified.             | n/a                    |
```

Comment:
Suggested change to description:

An array of whitelisted IP addresses for the Atlantis Service in Kubernetes.  If no value is specified, the Service will allow incoming traffic from all IP addresses (`0.0.0.0/0`).